RadarSDK/RadarGeofence.m

```diff
@@ -124,7 +124,7 @@ - (instancetype _Nullable)initWithObject:(id)object {
 
         if ([type isEqualToString:@"circle"]) {
             geometry = [[RadarCircleGeometry alloc] initWithCenter:center radius:radius];
-        } else if ([type isEqualToString:@"polygon"] || [type isEqualToString:@"isochrone"]) {
+        } else if ([type isEqualToString:@"polygon"] || [type isEqualToString:@"Polygon"] || [type isEqualToString:@"isochrone"]) {
```

Comment:
Are we seeing `"Polygon"` coming back from the API?

Reply:
Oh, I'm guessing we were thinking `GeoJSON` `type = "Polygon"`, which would be `geofence.geometry.type`, not `geofence.type`

Reply:
that's correct